sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Azure.Verticals.AgriFood.Farming.csproj

```diff
@@ -12,6 +12,10 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+  </ItemGroup>
+
+  <!-- These packages are included as part net6.0; the external references are not needed. -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
It would be better to use `Condition="!$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'net6.0'))"` for all such conditions, compatibility check is future-proof